sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/image/automl_image.py

```diff
@@ -45,14 +60,27 @@ def __init__(
 
     @property
     def log_verbosity(self) -> LogVerbosity:
+        """Returns the verbosity of the logger.
+        :return: The log verbosity.
+        :rtype: ~azure.ai.ml._restclient.v2023_04_01_preview.models.LogVerbosity
+        """
         return self._log_verbosity
 
     @log_verbosity.setter
     def log_verbosity(self, value: Union[str, LogVerbosity]):
+        """Sets the verbosity of the logger.
```

Comment:
displayed incorrectly in docs, add new line before paramlist

Reply:
added new line. I think, either it's an issue with UI if it get space, it displays in two line OR it's an issue with "output" property since it is empty, it gives that space for next elements to fill in same line.

For NLP also, I see the same rendering for output.
![image](https://github.com/Azure/azure-sdk-for-python/assets/107195344/62798886-f8c7-4953-863a-32e176045e7d)